docs/scenarios_authoring/TestingAndDebugging.md

```diff
@@ -18,7 +18,7 @@ Below you can see how such file looks like.
 {"sourceId":"kafka1","record":{"keySchemaId":null,"valueSchemaId":null,"consumerRecord":{"key":null,"value":{"clientId":"4","amount":30,"eventDate":1674548921},"topic":"transactions","partition":0,"offset":58209,"timestamp":1674548933921,"timestampType":"CreateTime","headers":{},"leaderEpoch":0}},"timestamp":1674548933921}
 {"sourceId":"kafka2","record":{"keySchemaId":null,"valueSchemaId":null,"consumerRecord":{"key":null,"value":{"clientId":"4","amount":30,"eventDate":1674548921},"topic":"transactions","partition":0,"offset":58209,"timestamp":1674548933921,"timestampType":"CreateTime","headers":{},"leaderEpoch":0}},"timestamp":1674548933921}
 ```
-Each line of this file represents the next ongoing event and specify which source it targets with `sourceId` field. The json representation of the event is in `record` field.
+Each line of the generated file contains one event. The `sourceId` field contains name of the source node as specified in the scenario and the `record` field contains json representation of the event. 
```

Comment:
1. "Each line of the generated file" -> "Each line in the generated file"
2. "contains name of the source node" -> "contains the name of the source node"
3. "field contains json representation" -> "field contains the JSON representation"
4. "the scenario and the `record`" -> "the scenario, and the `record`"